collect_app/src/main/java/org/odk/collect/android/tasks/FormLoaderTask.java

```diff
@@ -559,8 +561,9 @@ private void readCSV(File csv, String formHash, String pathHash) {
                 ida.addRow(pathHash, columnHeaders, nextLine);
 
             }
-        } catch (IOException e) {
-            Timber.e(e, "Exception thrown while reading csv file");
+        } catch (IOException | SQLiteException e) {
+            warrningMsg = "Exception thrown while reading csv file";
```

Comment:
Given this is going to get shown to the user, should we not move it into `strings.xml` so we can translate it? Also, Android Studio probably highlights this and is angry about it. It hates Strings.

Reply:
I was thinking about the same so agree.